src/main/java/com/upsolver/datasources/jdbc/JDBCDataSource.java

```diff
@@ -303,10 +309,22 @@ private ResultSet queryData(JDBCTaskMetadata metadata, int limit, Connection con
                     timestampCols,
                     fullLoad);
         } catch (SQLException e) {
-            return Collections.singletonList(new PropertyError(connectionStringProp, "Unable to connect to database, please ensure connection string and login info is correct.\n" +
-                    "SqlError: " + e.getMessage()));
+            Collection<SQLDriver> suitableDrivers = sqlDrivers.getDrivers().stream().filter(driver -> connectionString.startsWith(driver.getUrlPrefix())).collect(Collectors.toList());
+            final String errorMessage;
+            if (suitableDrivers.isEmpty()) {
+                logger.warn("Unable to connect to database, using JDBC URL: {}", connectionString, e);
```

Comment:
This isn't a warn log. We know what the issue is and it's a legitimate flow

Reply:
done